src/MainCase/MainCase.scad

```diff
@@ -266,8 +266,8 @@ module MainCase(without_inserts=false) {
             polygon(polyRound([
               [0, 0, 0],
               [UsbCharger_width, 0, 0],
-              [UsbCharger_width, UsbCharger_depth, 3],
-              [0, UsbCharger_depth, 0],
+              [UsbCharger_width, UsbCharger_depth+wall_thickness/2, 3],
```

Comment:
Otherwise the wall is not even 0.4mm wide and not printed.

Reply:
The wall there needs to be very thin, as it will prevent the USB-C plug from going in deep enough if it is too thick. Please make it a constant 0.8mm or so (instead of depending on wall_thickness). That will be 2 traces at 0.4mm nozzle, or one trace if printing with a wider nozzle. I don't think we can afford much more.